vllm/config.py

```diff
@@ -579,6 +579,10 @@ class CacheConfig:
             profiled num_gpu_blocks if specified. Does nothing if None.
     """
 
+    # Will be set after profiling.
+    num_gpu_blocks: Optional[int] = None
+    num_cpu_blocks: Optional[int] = None
```

Comment:
Doesn't this set the value at the class level rather than the instance level?

Reply:
oops ... not sure what I was thinking here.

Thank you for reviewing so closely! That's why I wanted to send it in smaller chunks to make sure it got a close look.